omero/sysadmins/server-upgrade.rst

```diff
@@ -58,7 +58,10 @@ generated::
 
 We recommend you run the command with ``--dry-run`` first to list the pyramids
 that will be deleted. If there are a large number of pyramids, you may need to
-run the command more than once. You can also specify a cut-off date (e.g. the
+run the command more than once since you cannot remove more than 500 pyramids in one call.
```

Comment:
you can, using `--limit` -- maybe add some qualifier about "by default" or something rather than explaining fully

Reply:
you won't be able to do more than 500 even if you increase the limit

Reply:
Ahh, help text didn't mention that detail! This is fine then.

Reply:
Could improve the help in a follow-up PR

Reply:
:+1: No urgency.